pulsar-discovery-service/src/main/java/com/yahoo/pulsar/discovery/service/server/DiscoveryServiceStarter.java

```diff
@@ -78,7 +78,7 @@ public void run() {
     protected static void startWebService(ServerManager server, ServiceConfig config) throws Exception {
         // add servlet
         Map<String, String> initParameters = new TreeMap<>();
-        initParameters.put("zookeeperServers", config.getZookeeperServers());
+        initParameters.put("localZookeeperServers", config.getLocalZookeeperServers());
```

Comment:
We've being using `zookeeperServers` elsewhere for the "local ZK connection", we should not change it just here